examples/ipynb/webgl_example_1.ipynb

```diff
@@ -189,21 +191,21 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 2",
```

Comment:
Yes, I understood that, but it clutters up the commit unnecessarily. Reverting would just add another unnecessary commit. You'd need to rebase to fix it correctly. In the future, you should use `git add -p` and `git diff`/`git diff --cached` to verify what you are adding to a commit.